apps/src/weblab2/Weblab2View.tsx

```diff
@@ -85,6 +101,29 @@ const Weblab2View: React.FC<
     state => !!state.lab2Project.projectSources?.source
   );
 
+  // Set up AI Tutor system prompt options based on available modes in level properties.
+  useEffect(() => {
+    const availableModes = levelProperties.availableAiTutorModes || [
+      DEFAULT_AI_TUTOR_MODE,
+    ];
+    const systemPromptName = getPromptNameFromMode(
+      availableModes ? availableModes[0] : undefined
```

Comment:
fyi (outside scope of this PR) - [asked](https://docs.google.com/document/d/1JJJo8yCTtA94yJPgsyBASN5FYudMMBj2r36IEOU-tks/edit?disco=AAABpFwLCNk) about whether levelbuilders want to be able to select the default mode if multiple modes available. In `aichat`, levelbuilders can select available models and default/initial model.